index.html

```diff
@@ -91,39 +91,28 @@
           </p>
           <p>
             Firecracker is used by/integrated with (in alphabetical order):
-            <a href="https://appfleet.com/">appfleet</a>
-            , containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            ,
-            <a href="https://fly.io">Fly.io</a>
-            ,
+            <a href="https://appfleet.com/">appfleet</a>,
```

Comment:
can we apply the same formatting rule for consistency to the code starting from line 270 please?